src/AbpHelper.Core/Templates/Crud/Groups/UiRazor/src/{{ProjectInfo.FullName}}.Web/Pages/{{Bag.PagesFolder}}/{{EntityInfo.RelativeDirectory}}/{{EntityInfo.Name}}/CreateModal.cshtml

```diff
@@ -8,12 +8,14 @@
 @{
     Layout = null;
 }
-<abp-dynamic-form abp-model="ViewModel" data-ajaxForm="true" asp-page="CreateModal">
+<form abp-model="ViewModel" data-ajaxForm="true" asp-page="CreateModal">
```

Comment:
I think these changes should roll back since removing the dynamic form is a big change.

Reply:
Also, `abp-input` cannot be applied to all types, such as `Enum` and `bool`.

Reply:
I do this because  of custom styling in big domains

Reply:
I like this change, but we should create a new PR to discuss and implement it since it is complicated and not directly related to this PR.

Reply:
We use small PRs to make small changes, so when we create a new release, it automatically generates a list to show people what changes we did.
![image](https://user-images.githubusercontent.com/30018771/171987037-f4ec7f18-1031-4a75-adf2-2a9cbc65c668.png)

Reply:
thanks